rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -542,14 +643,21 @@ async fn parse_v2_coin(
     let mut current_fungible_asset_balances = current_fungible_asset_balances
         .into_values()
         .collect::<Vec<CurrentFungibleAssetBalance>>();
+
     // Sort by PK
     fungible_asset_metadata.sort_by(|a, b| a.asset_type.cmp(&b.asset_type));
     current_fungible_asset_balances.sort_by(|a, b| a.storage_id.cmp(&b.storage_id));
 
+    // Process the unified balance
+    let current_unified_fungible_asset_balances = current_fungible_asset_balances
+        .iter()
+        .map(CurrentUnifiedFungibleAssetBalance::from)
+        .collect::<Vec<_>>();
```

Comment:
One thing to note is that we can't have partial fill for current_fungible_asset_balances b/c it'd override current_unified_fungible_asset_balances.

Reply:
What do you mean by partial fill?

Reply:
Let's say we have wallet A and we transfer some $$ there and the it gets deleted. We'd have partial write for the delete correct? But this logic would miss the delete I think. I guess it won't be overridden automatically since the deletes would be a different map.